blacklight-cornell/app/assets/stylesheets/cornell/_browse-info.scss

```diff
@@ -27,10 +20,17 @@ ul.nav-tabs li {
   margin-bottom: 16px;
 }
 .ld-acknowledge {
-    font-size: 14px;
+    margin-top: 2rem;
+    padding-top: 1rem;
+    border-top: 1px solid $tan;

```

Comment:
It looks like this is adding a small border to the top of the dbpedia acknowledgements too, in the middle of the dbpedia descriptions:
![Screenshot 2025-06-23 at 2 17 14 PM](https://github.com/user-attachments/assets/cec1f4fe-4268-46c2-a4e4-ec56fff647dd)

It's a little difficult to see!